recipe/meta.yaml

```
@@ -1,30 +1,36 @@
-{% set version = "1.21.4" %}
+{% set version = "1.24.5" %}
 
 package:
   name: wget
   version: {{ version }}
 
 source:
   url: https://ftp.gnu.org/gnu/wget/wget-{{ version }}.tar.gz
-  sha256: 81542f5cefb8faacc39bbbc6c82ded80e3e4a88505ae72ea51df27525bcde04c
-
+  sha256: fa2dc35bab5184ecbc46a9ef83def2aaaa3f4c9f3c97d4bd19dcb07d4da637de
+  # Patching for CVE is required. When the newest version is availble,
+  # this patch can be removed.
```

```suggestion
  # Patching for CVE-2024-38428 is required. Once a cleared version is released,
  # this patch can be removed.
```
Best to be a bit more specific to simplify verifying the fix later.

---

Thanks Melody! 😄